### PR TITLE
enhance kube-schedule init flag

### PR DESCRIPTION
**Pull request by zjj2wry**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
```
remove TODO: once we switch everything over to Cobra commands, we can go back to calling
utilflag.InitFlags() (by removing its pflag.Parse() call). 
For now, we have to set the normalize func and add the go flag set by hand.
```

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: plugin/cmd/kube-scheduler/app/server.go [hunk 1]
```diff
@@ -104,7 +104,7 @@ type Options struct {
 }
 
 // AddFlags adds flags for a specific SchedulerServer to the specified FlagSet
-func AddFlags(options *Options, fs *pflag.FlagSet) {
+func (options *Options) AddFlags(fs *pflag.FlagSet) {
```
Comment (bsalamat) [inline plugin/cmd/kube-scheduler/app/server.go]: s/options/o/  to be consistent with the rest of the methods.

**Comment by zjj2wry** [comment]
@bsalamat updated, ptal

**Comment by zjj2wry** [comment]
/assign @luxas
need get approve in kubeadm，thanks

**Comment by zjj2wry** [comment]
ping @timothysc @luxas @mikedanese PTAL for kubeadm change.

**Comment by zjj2wry** [comment]
cc @kubernetes/sig-kubeadm-pr-reviews

**Review by luxas — APPROVED** [review]
/approve

Sorry for the approval latency @zjj2wry 
Thanks for this PR :)

**Pull request merged**